### ISSUE TITLE
Geographic Raster failed validation creates an aggregation without metadata

### ISSUE THREAD

**Issue by sblack-usu** [hydroshare/hydroshare#4405]
**Description of the bug**
When a .tif file with incorrect longitude is uploaded, the auto-aggregation fails and leaves the aggregation in a dirty state without a metadata object.

Steps to reproduce the bug:
1. Download [NA_CORDEX_25yr_SWE_Mid_9model_ensemble_sd.tif.zip](https://github.com/hydroshare/hydroshare/files/7428294/NA_CORDEX_25yr_SWE_Mid_9model_ensemble_sd.tif.zip)
2. Upload the uncompressed tif file to a resource
3. Click on the resulting aggregation and observe the Nonetype object error, indicating the metadata is gone

**Expected behavior**
I'm not sure what should happen.  I'm unsure if the tif file is correct and there is something wrong with our validation.  However, if any of our validation fails, the aggregation should be reverted and the file should be a regular file.

**Additional information**
Check the validation to see if there is a bug in the validation and fix it.  Update all aggregations to revert if an error is encountered rather than create a dirty state with no metadata object.

```
[27/Oct/2021 17:29:18] ERROR [root:371] Geographic raster aggregation. Error when creating aggregation. Error:['Value for West longitude should be in the range of -180 to 180']
Traceback (most recent call last):
  File "/hydroshare/hs_file_types/models/raster.py", line 364, in set_file_type
    logical_file.metadata.create_element(k, **v)
  File "/hydroshare/hs_file_types/models/base.py", line 428, in create_element
    element = model_type.model_class().create(**kwargs)
  File "/hydroshare/hs_core/models.py", line 1398, in create
    cls.validate_coverage_type_value_attributes(kwargs['type'], value_arg_dict)
  File "/hydroshare/hs_core/models.py", line 1621, in validate_coverage_type_value_attributes
    raise ValidationError("Value for West longitude should be "
django.core.exceptions.ValidationError: ['Value for West longitude should be in the range of -180 to 180']
```

**Comment by pkdash** [comment]
@dtarb Form the example longitude ranges, it seems any value in the range of -360 to 360 is a valid value for longitude. Is that right?  Once it's a valid value then we need to adjust the value if needed based on the rules you have provided above.

**Comment by pkdash** [comment]
This has been fixed in 1.51.1 - so closing.
